requirements.txt

```diff
@@ -1,16 +1,27 @@
+commonmark==0.9.1
+h5py==2.10.0
+#ipykernel==5.1.3
+jupyter>=1.0.0,<2
```

Comment:
deepcell-tf has a separate requirements.txt for the documentation. Let's do that to declutter the main requirements file